cli/cli/commands/engine/engine.go

```diff
@@ -22,4 +23,5 @@ func init() {
 	EngineCmd.AddCommand(status.StatusCmd)
 	EngineCmd.AddCommand(stop.StopCmd)
 	EngineCmd.AddCommand(restart.RestartCmd)
+	EngineCmd.AddCommand(logs.EngineLogsCmd.MustGetCobraCommand())
```

Comment:
Why `MustGetCobraCommand` is here only for logs?

Reply:
The others are already native Cobra while logs uses the EngineConsumingKurtosisCommand interface